hotkey.js

```diff
@@ -14,12 +16,46 @@ function ignoreHotkeysIn(target) {
   )
 }
 
+function choseTheElement(current, selector) {
+  let list = current.querySelectorAll(selector)
+
+  for (let j = 0; j < list.length; j++) {
+    let closestElement = list[j].closest(`[${IGNORE_ATTR}]`)
+    if (closestElement === current || closestElement == null) {
+      return list[j]
+    }
+  }
+  return null
+}
+
 function checkHotkey(where, code, overrides) {
   let codeOverride = overrides[code]
   if (Object.values(overrides).includes(code) && !codeOverride) return false
-  return where.querySelector(`[aria-keyshortcuts="${codeOverride || code}" i]`)
+  let current = where.activeElement
+  let cssSelector = `[aria-keyshortcuts="${codeOverride || code}" i]`
+  let elementWithHotKey = null
+  
+  
+  if(choseTheElement(current, cssSelector) != null) {
+    elementWithHotKey = choseTheElement(current, cssSelector)
```

Comment:
You are passing the same selector to `choseTheElement`.

Just move it there and it will reduce JS bundle size